api/internal/git/repospec.go

```diff
@@ -131,37 +139,37 @@ func parseGitURL(n string) (
 		if len(n) > 0 && n[0] == '/' {
 			n = n[1:]
 		}
-		path, gitRef, gitTimeout, gitSubmodules = peelQuery(n)
+		path = parsePath(n)
 		return
 	}
 
 	if isLocal {
 		if idx := strings.Index(n, "//"); idx > 0 {
 			orgRepo = n[:idx]
 			n = n[idx+2:]
-			path, gitRef, gitTimeout, gitSubmodules = peelQuery(n)
+			path = parsePath(n)
 			return
 		}
-		path, gitRef, gitTimeout, gitSubmodules = peelQuery(n)
+		path = parsePath(n)
 		orgRepo = path
 		path = ""
 		return
```

Comment:
Why do we set `path = parsePath(n)` (L153) and then two lines later, reset to `path = ""` (L155)?

Seems like this would have the same effect:

```
orgRepo = parsePath(n)
path = ""
```

Reply:
Good catch! I didn't write this code, but I agree with and will add your suggested change.